AppServer/google/appengine/api/urlfetch_stub.py

```diff
@@ -420,11 +420,21 @@ def _RetrieveURL(url, payload, method, headers, request, response,
           if not _CONNECTION_SUPPORTS_TIMEOUT:
             socket.setdefaulttimeout(orig_timeout)
           connection.close()
-      except (_fancy_urllib_InvalidCertException,
-              _fancy_urllib_SSLError), e:
+      except _fancy_urllib_InvalidCertException, e:
         raise apiproxy_errors.ApplicationError(
           urlfetch_service_pb.URLFetchServiceError.SSL_CERTIFICATE_ERROR,
           str(e))
+      except _fancy_urllib_SSLError, e:
+
+
+
+
+
```

Comment:
Haha, it's the SDK code. I preserved the useless lines to reduce the diffs when comparing. It makes upgrading to new SDKs easier.